process_report/invoices/invoice.py

```diff
@@ -78,7 +82,14 @@ def _prepare_export(self):
         that should or should not be exported after processing."""
         pass
 
+    def _filter_columns(self):
+        """Filters and renames columns before exporting"""
+        self.data = self.data[self.export_columns_list].rename(
```

Comment:
This should NOT rename the dataframe at `self.data.`.

Reply:
From the Pandas documentation, `rename()` defaults to creating a copy of the dataframe. Though I have explicitly used `copy()` to make the intent clear.

Reply:
Regardless, the `self.data` attribute should point to the reference of original Dataframe that the Invoice was instantiated with. Do not reassign this `self.data`.

Reply:
I have applied your suggestion, and also added the `export_columns_list` for the PI-specific invoice, which I forgot to add while rebasing.